tests/integration/components/list-filter-test.js

```diff
@@ -31,10 +31,10 @@ module('Integration | Component | list-filter', function(hooks) {
       </ListFilter>
     `);
 
-    return settled().then(() => {
-      assert.equal(this.element.querySelectorAll('.city').length, 3);
-      assert.dom(this.element.querySelector('.city')).hasText('San Francisco');
-    });
+    await settled();
```

Comment:
Settled is not needed here (we have already awaited the `render` call just above).